playbooks/local_os_client.yaml

```diff
@@ -16,6 +16,33 @@
     set_fact:
       cloudsyaml: "{{ cloudsyaml['content'] | b64decode | from_yaml }}"
 
+  - name: Grab the CA certificate
+    when: ssl_enabled
+    block:
+      - name: Read CA certificate
+        slurp:
+          src: "{{ ssl_ca_cert_path }}"
+        register: ssl_ca_cert_output
+      - name: Set fact for CA cert
+        set_fact:
+          ssl_ca_cert: "{{ ssl_ca_cert_output['content'] | b64decode }}"
+      - name: Copy CA cert into PKI
```

Comment:
Hmmm, can we not do that, or at least make it optional? We don't need the certificate in the host trust bundle starting from OCP 4.7 and this is probably something we want to test. I also don't want to pollute my local trust bundle.

We should instead add the path to the CA Cert  in the generated `clouds.yaml` entry with the `cacert` key.

Reply:
I agree we should put this in clouds.yaml by preference. However, I suspect we'll need it in the trust bundle for some use cases anyway. e.g. Do we have openstack cli support for manila?

In short: if this were the only thing holding this patch up I'd merge it.

Reply:
@mandre FYI the `cacert` entry in clouds.yaml is already added when SSL is [configured in TripleO](https://opendev.org/openstack/tripleo-heat-templates/src/branch/master/deployment/keystone/keystone-container-puppet.yaml#L777-L781); so we don't need to take care of it.

So if I understand the suggestion, the ask is to make this step optional; so we can easily enable it but disable it by default. If that's the case, I agree and I'll do it.

Reply:
We'll presumably need to rewrite the cacert path in clouds.yaml for both the standalone and standalone_openshift users.

Reply:
> We'll presumably need to rewrite the cacert path in clouds.yaml for both the standalone and standalone_openshift users.

why that?

Reply:
the content is already good, I checked when testing. I didn't have to change anything manually.

Reply:
We should probably store the cert to a local directory owned by the user rather than requiring password-less sudo. In that case we would need to update the cacert path as Matt noted.

Reply:
I honestly have mixed feelings about this request, just because if the user finally decides to update their PKI with `update-ca-trust extract`, the CA cert needs to be in /etc/pki, not somewhere in $HOME.

Maybe we can improve this later?